### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
Add support for Python 3.9

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,25 +1,70 @@
...
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
-    zope-testrunner --test-path=src []
+    zope-testrunner --test-path=src {posargs:-vc}
 extras =
     test
     zcml
-    extract
```
Comment (sallner) [inline tox.ini:11]: `extract` currently only excludes `zope.tal` which is also included in `test`, so I understand, why it is missing here now. On the other hand it seems reasonable to me that we check all extras to be installable.

File: tox.ini [hunk 2]
```diff
@@ -1,25 +1,70 @@
...
 
-[testenv:flake8]
-basepython = python3.6
+[testenv:lint]
+basepython = python3
 skip_install = true
-deps = flake8
-commands = flake8 --doctests src testproj setup.py
```
Comment (sallner) [inline tox.ini:17]: We included the `testproj` folder for linting in the old setup. Could we somehow bring that back? Maybe via the configuration section?

**Review by sallner — APPROVED** [review]
Maybe give a thought about checking the `testproj` but that is not as big of a deal.

File: tox.ini (hunk 1 above)
Comment (icemac) [reply tox.ini:11]: `pip` ignores unknown extras, it only prints a warning, see the following example using `RestrictedPython`:

```
$ bin/pip install "RestrictedPython[i-do-not-exist]"
Collecting RestrictedPython[i-do-not-exist]
  Using cached RestrictedPython-5.1-py2.py3-none-any.whl (27 kB)
  WARNING: RestrictedPython 5.1 does not provide the extra 'i-do-not-exist'
Installing collected packages: RestrictedPython
Successfully installed RestrictedPython-5.1
```

But I am going to add the extra as well.

File: tox.ini (hunk 2 above)
Comment (icemac) [reply tox.ini:17]: Fixed in https://github.com/zopefoundation/zope.app.locales/pull/13/commits/db2f0b4043b0eaeb083e41151db212b648a19e9c

**Comment by icemac** [comment]
After https://github.com/zopefoundation/meta/pull/71 is merged this PR has to be adapted to use the new configuration option to define additional extras.

**Comment by sallner** [comment]
It seems that the alternative https://github.com/zopefoundation/meta/pull/76 is merged now.

File: .meta.toml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+with-sphinx-doctests = false
+
+[coverage]
+fail-under = 84
+
+[tox]
+testenv-additional = [
+    "    zcml",
+    "    extract",
+    ]
```
Comment (mgedmin) [inline .meta.toml:21]:
```suggestion
testenv-additional-extras = [
    "zcml",
    "extract",
    ]
```

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**